Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Control.cs

```diff
@@ -42,7 +52,27 @@ public float getValue()
             return value;
         }
 
-        public virtual void Start()
+        /// <summary>
+        /// Returns the current value mapped onto a range between 0% and 100%.
+        /// </summary>
+        /// <returns>Normalized Value</returns>
+        public float getNormalizedValue()
+        {
+            return Mathf.Abs(Mathf.Round((value - cvr.controlMin) / (cvr.controlMax - cvr.controlMin) * 100));
+        }
+
+        public void SetContent(GameObject content, bool hideContent)
```

Comment:
I know we don't document the VRTK_Control file. But I'm thinking maybe we should (like we document the WorldPointer which is an abstract class). Then all of these public methods can be documented (as well as the public event).

Then in each of the controls we can just say they extend this class (like the SimplePointer and BezierPointer docs show)

I think that would be more consistent and means (as I mentioned in the other comment) that there isn't the need to copy and paste the same docs between all of the controls